pkg/proxy/iptables/proxier.go

```diff
@@ -168,11 +190,34 @@ type endpointsInfo struct {
 	chainName utiliptables.Chain
 }
 
-// IPPart returns just the IP part of the endpoint.
+// IsLocal is part of interface.
```

Comment:
Explicitly state the interface in your comment. Please do it for the other commented functions as well.

Reply:
Done